mhs/mhs/common/state/work_description.py

```diff
@@ -14,7 +15,7 @@ class MessageStatus:
 DATA_KEY = 'MESSAGE_KEY'
 VERSION_KEY = 'VERSION'
 CREATED_TIMESTAMP = 'CREATED'
-LAST_MODIFIED_TIMESTAMP = 'TIMESTAMP'
+LATEST_TIMESTAMP = 'TIMESTAMP'
```

Comment:
Ah, what I actually meant by https://github.com/nhsconnect/integration-adaptors/pull/48#discussion_r311479442 was the string part.